static/js/server_events_dispatch.js

```diff
@@ -348,6 +348,12 @@ exports.dispatch_normal_event = function dispatch_normal_event(event) {
         ui.remove_message(msg_id);
         break;
 
+    case 'attachments':
+        if (event.op === 'add') {
+            page_params.attachments.push(event.attachment);
+        }
+        break;
```

Comment:
Is this going to add the new attachment on the right end of the list?  I think you might need to do something to interact with the `list_render` logic to get the UI to update. 

Also, I think you need a "remove" case.

A good way to test this sort of thing is to open 2 browsers, and have one interact with Zulip uploading and removing attachments while the second has the attachments UI open.  What you're looking for is the second one updating immediately when you make changes in the first.

Reply:
@timabbott 
I have added code to render list again.
Can you explain what do you mean by remove case because delete attachment is already being handled.